src/components/SnapshotAccessRequestView.test.tsx

```diff
@@ -25,24 +25,18 @@ const initialState = {
 };
 
 describe('SnapshotAccessRequestView', () => {
-  it('Renders an unordered list of snapshot access request ids', () => {
+  it('Renders', () => {
```

Comment:
For now there is very little testing we can do here - it is mostly a passthrough for state. It may grow, though, as we end up needing to do more searching and filtering serverside.